xmipp

```diff
@@ -806,40 +813,46 @@ def checkConfig():
             configDict["CUDA"]="False"
         if not checkMatlab(configDict):
             print(red("Cannot compile with Matlab, continuing without Matlab"))
-            configDict["MATLAB"]="False"
+            config_dict["MATLAB"]="False"
```

Comment:
configDict and config_dict don't exist in this scope.

Reply:
They are (from global scope of main) 
See e.g. https://stackoverflow.com/a/4775596/5484355